modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -2088,6 +2222,90 @@ private void _testPostObjectDefinitionWithSystemAggregationObjectField()
 				aggregationObjectFieldName));
 	}
 
+	@TestInfo("LPD-63539")
+	private void _testPostObjectDefinitionWithWorkflowDefinitionLink()
+		throws Exception {
+
+		// Company Scope
+
+		WorkflowDefinition workflowDefinition1 =
+			_workflowDefinitionManager.getWorkflowDefinition(
+				WorkflowDefinitionConstants.
+					EXTERNAL_REFERENCE_CODE_SINGLE_APPROVER,
+				TestPropsValues.getCompanyId());
+
+		WorkflowDefinitionLink workflowDefinitionLink1 =
+			new WorkflowDefinitionLink() {
+				{
+					groupExternalReferenceCode = StringPool.BLANK;
+					workflowDefinitionName = workflowDefinition1.getName();
+				}
+			};
+
+		WorkflowDefinitionLink[] workflowDefinitionLinks = {
+			workflowDefinitionLink1
+		};
+
+		ObjectDefinition objectDefinition = randomObjectDefinition();
+
+		objectDefinition.setWorkflowDefinitionLinks(workflowDefinitionLinks);
+
+		_assertWorkflowDefinitionLinks(
+			workflowDefinitionLinks, _addObjectDefinition(objectDefinition));
+
+		// Site scope
+
+		objectDefinition = randomObjectDefinition();
+
+		objectDefinition.setScope(ObjectDefinitionConstants.SCOPE_SITE);
```

Comment:
can we move this closer to `objectDefinition.setWorkflowDefinitionLinks(workflowDefinitionLinks);` ?